09-appendix-backends.Rmd

```diff
@@ -36,65 +40,83 @@ If you are using the Dockstore CLI, [follow these instructions](https://docs.doc
 
 ## Backend-specific notes
 
 ### HPCs
+
 It is difficult to provide specific advice on HPCs, as they can vary greatly. Some general notes:
 
-* Some HPCs do not support the use of Docker due to security concerns
+-   Some HPCs do not support the use of Docker due to security concerns
 
-  * HPCs that do not allow the use of Docker may be able to run WDLs using alternative container technologies such as podman or rootless Docker
+    -   HPCs that do not allow the use of Docker may be able to run WDLs using alternative container technologies such as podman or rootless Docker
 
-* Some HPCs will use `disks` to determine which disk to run on, which can be useful for managing disk space
+-   Some HPCs will use `disks` to determine which disk to run on, which can be useful for managing disk space
 
 #### Fred Hutch HPC
-* The Fred Hutch HPC supports the use of Docker, so the `docker` runtime attribute works as expected
 
-* The `memory` and `gpu` runtime attributes are ignored, but `cpu` works as expected
+-   [PROOF](https://proof.fredhutch.org/) is a Shiny frontend that launches Cromwell Server and lets you manage your jobs from a GUI. User guide can be found [here](https://sciwiki.fredhutch.org/dasldemos/proof-how-to/).
+
+-   The Fred Hutch HPC supports the use of multiple JSON files going into the same workflow
+
+-   The Fred Hutch HPC supports the use of Docker, so the `docker` runtime attribute works as expected
+
+-   `memory` and `gpu` runtime attributes are ignored, but `cpu` works as expected. Here is [some information](https://sciwiki.fredhutch.org/scicomputing/compute_jobs/#memory) on approximating memory use.
+
+-   `walltime` attribute is a string ("HH:MM:SS") that specifies how much time you want to request for the task. Can also specify \>1 day, e.g. "1-12:00:00" is 1 day+12 hours.
 
-* The Fred Hutch HPC supports the use of multiple JSON files going into the same workflow
+-   `partition` attribute specifies the [cluster partition](https://sciwiki.fredhutch.org/compdemos/gizmo_partition_index/)to use. The default is "campus-new".
+
+-   `modules` attribute is a a space-separated list of the [environment modules](https://sciwiki.fredhutch.org/scicomputing/compute_environments/) you'd like to load (in that order) prior to running the task. For example, "modules: "GATK/4.2.6.1-GCCcore-11.2.0 SAMtools/1.16.1-GCC-11.2.0", the GATK module will be loaded first, followed by the SAMtools module.
+
+-   `dockerSL` attribute allows users to use Docker and softlinks only to specific locations on the FH HPC Scratch disk. It is helpful when working with very large files. An example is: "ubuntu:latest". Just like the `docker` attribute, you should only specify this if you want the task to run in a container (otherwise the default will be a non-containerized job).
```

Comment:
This says "specific locations" but the example given looks like just a Docker string, not a specific location. Is this intentional?

Or do you mean that setting this variable means you can access specific locations as specified in the body of a workflow task?

Reply:
No idea. Got it from @sitapriyamoorthi's writing - can you help us out?

I can't find this attribute on FH HPC's official documentation, so I'm a bit tempted to leave this out.

Reply:
Put it in issue https://github.com/fhdsl/WDL_Workflows_Guide/issues/49